scripts/superdesk-search/views/item-search.html

```diff
@@ -19,23 +19,38 @@
     </fieldset>
     <fieldset ng-show="repo.search === 'local'">
         <div class="field">
-            <label for="search-headline">
-                {{:: 'Headline' | translate}}
+            <label for="search-slugline">
+                {{:: 'Slugline' | translate}}
             </label>
             <div class="control">
-                <input type="text" id="search-headline" ng-model="meta.headline">
+                <input type="text" id="search-slugline" ng-model="meta.slugline">
             </div>
         </div>
-
         <div class="field">
-            <label for="search-slugline">
-                {{:: 'Slugline' | translate}}
+            <label for="search-subject">
```

Comment:
just arranged in similar order that appears in header section of authoring, slugline > subject > headline and so on.